Framework/DataHandling/inc/MantidDataHandling/GroupDetectors2.h

```diff
@@ -239,8 +241,8 @@ class DLLExport GroupDetectors2 : public API::Algorithm {
     /// spectrum number to the this
     EMPTY_LINE = 1001 - INT_MAX, ///< when reading from the input file this
     /// value means that we found any empty line
-    IGNORE_SPACES = Poco::StringTokenizer::TOK_TRIM ///< equal to
-    /// Poco::StringTokenizer::TOK_TRIM but
+    IGNORE_SPACES = Mantid::Kernel::StringTokenizer::TOK_TRIM ///< equal to
+    /// Mantid::Kernel::StringTokenizer::TOK_TRIM but
     /// saves some typing
   };
 
```

Comment:
I did not spot this before, but looking closer it seems these two instances of `IGNORE_SPACES` now mean different things, while they meant the same before.

Was this uncovered by a test?